mav_radius: 0.3
max_v: 1.5
max_a: 2.5
max_yaw_vel: 1.5707
max_yaw_acc: 1.5707
